test/conformance/image/go-runner/main.go

```diff
@@ -59,6 +59,12 @@ func configureAndRunWithEnv(env Getenver) error {
 	// Print the output to stdout and a logfile which will be returned
 	// as part of the results tarball.
 	logFilePath := filepath.Join(resultsDir, logFileName)
+	// ensure the resultsDir actually exists
+	if _, err := os.Stat(resultsDir); os.IsNotExist(err) {
+		if mkdirErr := os.Mkdir(resultsDir, 0755); mkdirErr != nil {
```

Comment:
nit: I'd make a log line showing you're creating this directory so we know we hit that code path. If there are errors in how someone sets/reads resultsDir, they may be not realizing they are hitting this.

Reply:
yes, good suggestion, it would be better adding a log print before create the folder.
I suppose a log  line like
log.Printf("The resultsDir %v does not exist, will create it", resultsDir)